samples/bluetooth/peripheral_hr_coded/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _peripheral_hr_coded:
+
+Bluetooth: Peripheral Heart Rate Monitor with Coded PHY
+#######################################################
+
+Overview
+********
+
+Similar to the :ref:`zephyr:peripheral_hr` sample, except that this application specifically uses LE Coded PHY for communication.
+Once a device connects, it generates dummy heart rate values.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set23_start
+   :end-before: set23_end
+
+
+Building and Running
+********************
+.. |sample path| replace:: :file:`samples/bluetooth/peripheral_hr_coded`
+
+.. include:: /includes/build_and_run.txt
+
+
+Testing
+=======
+
+After programming the sample to your board, you can test it by connecting to another board that is running the :ref:`central_hr_coded`.
+
+Testing with another board
+--------------------------
+
+1. |connect_terminal_specific|
+#. Reset the board.
+#. Program the other board with the :ref:`central_hr_coded`
```

Comment:
Documentation build is failing because the `central_hr_coded` label does not exist.

Reply:
yes, this is fixed now